[libxlsxwriter] upgrade to 0.8.7

jmcnamara opened the pull request:
Upgrade to the latest version of libxlsxwriter.

Closes #6982

NNemec committed:
mesonbuild - Update to 0.51.0

NNemec committed:
set cmake_prefix_path correctly within vcpkg_configure_meson

jmcnamara committed:
[libxlsxwriter] upgrade to 0.8.7

Upgrade to the latest version of libxlsxwriter.

Closes #6982

jmcnamara committed:
[libxlsxwriter] upgraded to 0.8.7

Upgrade to the latest version of libxlsxwriter.

Closes #6982

msftclas commented:
[![CLA assistant check](https://cla.opensource.microsoft.com/pull/badge/not_signed)](https://cla.opensource.microsoft.com/microsoft/vcpkg?pullRequest=6992) <br/>Thank you for your submission, we really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/vcpkg?pullRequest=6992) before we can accept your contribution.<br/><br/>:x: jmcnamara [sign now](https://cla.opensource.microsoft.com/microsoft/vcpkg?pullRequest=6992)<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla.opensource.microsoft.com/check/microsoft/vcpkg?pullRequest=6992) it.</sub>

Rastaban committed:
[ppconsul] remove conflict with json11 (#6967)

* [ppconsol] remove conflict with json11

jmcnamara commented:
Could someone explain the build failures to me.

JackBoosY committed:
[flann]Upgrade version to 1.9.1 and fix build error. (#6931)

* [flann]Upgrade version to 1.9.1 and fix build error.

* [flann]apply new patches.

* [flann]Regenerate patches.

* [flann]Combine all patches.

JackBoosY committed:
[libtins]Upgrade version to 4.2 and adds dependent ports to new version. (#7008)

vicroms committed:
[--overlay-ports] Show location of overriden ports during install plan (#7002)

* [--overlay-ports] Show source location of overlayed ports during install plan

* Code cleanup

* Code cleanup

myd7349 committed:
[sprout] Add new port (#6997)

myd7349 committed:
[doctest] Update to 2.3.3 (#6998)

grdowns committed:
Merge pull request #6977 from NNemec/mesonbuild-cmake_prefix_path

set cmake_prefix_path correctly within vcpkg_configure_meson

cenit committed:
[darknet] update to latest release (#6787)

* [darknet] update to latest release

* [darknet] Reduce patch size

* update to latest release, does not require any patch :)

bsergean committed:
[ixwebsocket] add new port (#6835)

* Add ixwebsocket

* remove commented cmake directives

* add dependencies

* directive to remove share folder

* try again with 4.0.3 which is using find_library for external code instead of vendoring it by default

* use vcpkg feature flag to detect ssl backend + compile ws

* do not build ws command line tool

* add a feature for osx / control file closer to curl one

* stop using features, as this makes CI unhappy

* Revert "stop using features, as this makes CI unhappy"

This reverts commit 4666b185671a9e37de5252334965fd63c060aea0.

* play again with Build-Depends to fix error which appears to be only on uwp

* call vcpkg_from_github instead

* share folder not created

tarcila committed:
[scripts] Fix vcpkg_fixup_cmake on non Windows platforms (#5630)

* [scripts] Fix vcpkg_fixup_cmake on non Windows platforms

Script was only handling tools executables ending with .exe.
Changed it so anything under /bin/ in transformed.

This fixes for instance FlatcTargets-release.cmake from flatbuffers
port on osx.

Rastaban committed:
[cnl] Update cnl to latest(#7031)

LilyWangL committed:
[armadillo] Fix build error in Linux (#7022)

 [armadillo] Fix installation path

tarcila committed:
[capnproto] Enable Linux and OSX support (#7024)

Now that #5630 is fix, this package is fully supported on both Linux
and Mac OS X.

LilyWangL committed:
[nana, fmi4cpp] Fix Visual Studio 2019 deprecates <experimental/filesystem>. (#7021)

NancyLi1013 committed:
[libpmemobj-cpp] Add new port (#7020)

JackBoosY committed:
[qt5-base]Add execute permission when installing executables in Linux. (#7019)

jmcnamara committed:
Merge branch 'master' of https://github.com/jmcnamara/vcpkg
